test/testcafe/spec/IdentifyRegistration_spec.js

```diff
@@ -175,3 +178,20 @@ test.requestHooks(enrollProfileFinishMock)('should show terminal screen after re
 
   await t.expect(registrationPage.getTerminalContent()).eql('An activation email has been sent to john@gmail.com. Follow instructions in the email to finish creating your account');
 });
+
+test.requestHooks(mock)('should call settings.registration.click on "Sign Up" click, instead of moving to registration page', async t => {
+  const identityPage = new IdentityPageObject(t);
+  await identityPage.navigateToPage();
+  await rerenderWidget({
+    baseUrl: 'http://localhost:3000',
+    registration: {
+      // eslint-disable-next-line
+      click: () => console.log('hello')
```

Comment:
be more explicit on the message to make sure this message is coming for this test case.

Reply:
Can we update this message to something like `console.log('registration click handler fired')` ?